packages/pieces/community/pinterest/src/lib/actions/update-board.ts

```diff
@@ -0,0 +1,45 @@
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { makeRequest } from '../common';
+import { pinterestAuth } from '../common/auth';
+import { HttpMethod } from '@activepieces/pieces-common';
+import { boardIdDropdown } from '../common/props';
+
+
+export const updateBoard = createAction({
+  auth: pinterestAuth,
+  name: 'updateBoard',
+  displayName: 'Update Board',
+  description: 'Modify a board’s name, description, or visibility settings.',
+  props: {
+    board_id: boardIdDropdown,
```

Comment:
Missing Ad Account Support: No support for ad_account_id query parameter:
```suggestion
       props: {
     ad_account_id: adAccountIdDropdown, // Add this
     board_id: boardIdDropdown,
     // ... other props
   }
```